frontend/Dockerfile

```diff
@@ -20,7 +20,6 @@ ENV HOME=/tmp \
 COPY --from=build /near-explorer/production_node_modules ./node_modules
 COPY --from=build /near-explorer/frontend/public ./public
 COPY --from=build /near-explorer/frontend/next.config.js ./
-COPY --from=build /near-explorer/frontend/src/libraries/consts.js ./src/libraries/consts.js
```

Comment:
@frol 
I removed `consts.js` from here, it seems like it's pretty much unneeded now